examples/jupyter/demo002_diffusion.ipynb

```diff
@@ -15,9 +17,9 @@
    "source": [
     "# dependencies\n",
     "using Pkg\n",
-    "Pkg.activate(\"./\")\n",
+    "Pkg.activate(\".\")\n",
     "Pkg.instantiate()\n",
-    "Pkg.develop(path=\"./../../../LFAToolkit.jl\")\n",
+    "Pkg.develop(path=\"../..\")\n",
```

Comment:
I think it's better to not require that the clone be named exactly this. The simpler form works for me.